### ISSUE TITLE
The sampling of colours is different

### ISSUE THREAD

**Issue by thvi** [ThePacielloGroup/CCAe#278]
Hello,
Colour Contrast Analyser (CCA) v. 3.1.2 doesn't work correctly
Best.

**Comment by ferllings** [comment]
I removed your link for security purpose, as the target was not clearly identifiable.
Please attach an image instead.

**Comment by diprosew** [comment]
I agree with thvi, the eyedropper sampler results are incorrect.
Proof in point: Type in 100% red HEX value into the input field, i.e. #FF0000, then sample the displayed color and the resultant sample is not the same. The rgb values say it all; The result is rgb(247, 0, 12).
100% green/rgb(0, 255, 0) when sampled becomes rgb(72, 254, 0)

**Comment by ferllings** [comment]
The result may vary according to your screen color profile. See #76

**Comment by thvi** [comment]
![ezgif-5-94902427ac](https://user-images.githubusercontent.com/6812634/219300463-b7dbae81-4746-40a7-b54d-d9ddeea56382.gif)

**Comment by thvi** [comment]
<img width="1680" alt="Screenshot 2023-02-16 at 8 57 18 AM" src="https://user-images.githubusercontent.com/6812634/219303118-f28eb32c-679f-45b5-9391-29b9ce3d0e21.png">

**Comment by ferllings** [comment]
That's what I'm saying, if figma and CCA are not in the same color space, you may have different results

**Comment by thvi** [comment]
> That's what I'm saying, if figma and CCA are not in the same color space, you may have different results

So, how can I resolve it?

**Comment by ferllings** [comment]
As described in #76, that's not an easy problem. (If you application allows it) You need everything in the same color space
